app/models/cfe/remarks.rb

```diff
@@ -36,17 +38,29 @@ def populate_review_transactions
 
     def populate_review_reasons
       reasons = []
-      @hash.each do |_category, reason_hashes|
-        reasons += reason_hashes.keys
+      @hash.each do |category, reason_hashes|
+        reasons += case reason_hashes
+                   when Array
+                     reasons << category
```

Comment:
does reasons << category  return a duplication? I would assume just returning category would suffice

because to me the code is adding to the reasons array with << and the case is returning that value to reasons +=
Thus reaons will be [category1, category1]

Reply:
I think there was only ever one category with the new remark type so, for now, it will only ever be `[category]`
But may need addressing in the future if more types are added 🤔